canonical-web-and-design/vanilla-framework
# Detached tooltip

## Done

Added a detached tooltip example

Fixes https://github.com/canonical-web-and-design/vanilla-squad/issues/1000

## QA

- Open [demo](https://vanilla-framework-3677.demos.haus/docs/examples/patterns/tooltips/detached)
- See that the tooltip is visible when hovering over the button
- Check updated documentation:
  - [Tooltips](https://vanilla-framework-3677.demos.haus/docs/patterns/tooltips)
  - [Component status](https://vanilla-framework-3677.demos.haus/docs/component-status)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
